test/web/10-VerifiableCredentialStore.js

```diff
@@ -115,7 +115,8 @@ describe('VerifiableCredentialStore', () => {
     should.not.exist(credential);
   });
 
-  it('should query for an AlumniCredential with any issuer', async () => {
+  it('should not find credential when querying for an AlumniCredential' +
```

Comment:
```suggestion
  it('should not find credential when querying for an AlumniCredential ' +
```

Reply:
Shouldn't we keep the existing test and *add* this new one?

Reply:
I am not sure what you mean. There is already a test for the positive case where we are passing the correct issuer in the query here ---> https://github.com/digitalbazaar/bedrock-web-vc-store/pull/16/files#diff-affd32a517d78f0385d37ce90c3642cfbd3233ce09cbd12ab2f5246e1a132d57R140. This one on L118 is the test for the negative case.

Reply:
Well, there's a test for one with a correct issuer, but this test used to say "any issuer" -- where `trustedIssuer` is not set. There are three cases to check:

1. Match on `trustedIssuer`.
2. Miss on non-matching `trustedIssuer`.
3. Match on no `trustedIssuer` (any will do).

Reply:
Added test for the `match on no trustedIssuer` case here f0cd1d0